src/Stratis.Bitcoin.Features.SmartContracts/PoA/IFullNodeBuilderExtensions.cs

```diff
@@ -37,6 +38,35 @@ public static IFullNodeBuilder UseSmartContractPoAConsensus(this IFullNodeBuilde
             return fullNodeBuilder;
         }
 
+        /// <summary>
+        /// Configures the node with the smart contract proof of authority consensus model.
+        /// </summary>
+        public static IFullNodeBuilder UseSignedContractPoAConsensus(this IFullNodeBuilder fullNodeBuilder)
```

Comment:
This is unnecessary, why not use the extension method I created in 32e8b93?

Reply:
That method would also have to remove the existing rule, I thought it was pretty grotty to be picking at consensus rules there